webapp/src/modules/game/game.redux.js

```diff
@@ -18,15 +18,22 @@ export const { Types: GameTypes, Creators: GameActions } = createActions({
   pokeAnimal: ['fieldIndex'],
   buyAnimal: ['fieldIndex', 'cost'],
   upgradeWarehouse: ['fieldIndex', 'cost'],
+  hideInstruction: null,
 }, { prefix: 'GAME_' });
 
 export const GameRecord = new Record({
   data: Map(),
+  userLoggedIn: fromJS(false),
```

Comment:
For sure naming of this property is right in regarding that it is controlling of showing instruction modal, nothing more.